website/content/en/docs/setup/installation/package-managers/homebrew.md

```diff
@@ -9,7 +9,7 @@ weight: 4
 ## Installation
 
 ```shell
-brew tap timberio/brew && brew install vector
+brew tap vectordotdev/brew && brew install vector
```

Comment:
I wonder if this was actually changed?

Reply:
Yeah, this should be updated. `timberio/brew` still works due to Github redirecting it.

Reply:
Cool, not familiar enough with brew taps to know exactly how that works 😆